debian/copyright

```diff
@@ -0,0 +1,507 @@
+shall not be subject to revocation, rescission, cancellation, termination, or
+any other legal or equitable action to disrupt the quiet enjoyment of the Work
+by the public as contemplated by Affirmer's express Statement of Purpose.
+
+3. Public License Fallback. Should any part of the Waiver for any reason be
+judged legally invalid or ineffective under applicable law, then the Waiver
+shall be preserved to the maximum extent permitted taking into account
+Affirmer's express Statement of Purpose. In addition, to the extent the Waiver
+is so judged Affirmer hereby grants to each affected person a royalty-free, non
+transferable, non sublicensable, non exclusive, irrevocable and unconditional
+license to exercise Affirmer's Copyright and Related Rights in the Work (i) in
+all territories worldwide, (ii) for the maximum duration provided by applicable
+law or treaty (including future time extensions), (iii) in any current or
+future medium and for any number of copies, and (iv) for any purpose
+whatsoever, including without limitation commercial, advertising or promotional
+purposes (the "License"). The License shall be deemed effective as of the date
+CC0 was applied by Affirmer to the Work. Should any part of the License for any
+reason be judged legally invalid or ineffective under applicable law, such
+partial invalidity or ineffectiveness shall not invalidate the remainder of the
+License, and in such case Affirmer hereby affirms that he or she will not (i)
+exercise any of his or her remaining Copyright and Related Rights in the Work
+or (ii) assert any associated claims and causes of action with respect to the
+Work, in either case contrary to Affirmer's express Statement of Purpose.
+
+4. Limitations and Disclaimers.
+
+    No trademark or patent rights held by Affirmer are waived, abandoned,
+surrendered, licensed or otherwise affected by this document.  Affirmer offers
+the Work as-is and makes no representations or warranties of any kind
+concerning the Work, express, implied, statutory or otherwise, including
+without limitation warranties of title, merchantability, fitness for a
+particular purpose, non infringement, or the absence of latent or other
+defects, accuracy, or the present or absence of errors, whether or not
+discoverable, all to the greatest extent permissible under applicable law.
+Affirmer disclaims responsibility for clearing rights of other persons that may
+apply to the Work or any use thereof, including without limitation any person's
+Copyright and Related Rights in the Work. Further, Affirmer disclaims
+responsibility for obtaining any necessary consents, permissions or other
+rights required for any use of the Work.  Affirmer understands and acknowledges
+that Creative Commons is not a party to this document and has no duty or
+obligation with respect to this CC0 or use of the Work.
+
+Licence: BSD-MIT
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Added BSD-MIT,
After taking a look at the BSD_MIT license we note that it is a MIX of BSD-2-clause and MIT, in debian I can't find a keyword for this license but mentioned in page [1]
	 "There are many versions of the MIT license. Please use Expat instead,
	 when it matches.".

The ccan/ LICENSE.BSD-MIT is same as Expact license, see Expat license[2].

Already there are a files that licensed under BSD-2-clause, so there is also a BSD-2-clause as is without mixing with MIT in the copyright file.

When we use license that not listed under debian keywords lintain throw error [3].
Could you please assist me to find the right solution for this issue?
What is the keyword name should we use for those licenses ? 

[1]
https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-specification

[2]
http://www.jclark.com/xml/copying.txt

[3]
rdma-core source: invalid-short-name-in-dep5-copyright bsd